packages/pwa-kit-create-app/assets/pwa/pwa-kit.config.json

```diff
@@ -0,0 +1,25 @@
+{
+    "app": {
+        "defaultSiteId": "RefArchGlobal",
+        "url": {
+            "locale": "none",
+            "site": "none"
+        },
+        "sites": [
+            {
+                "id": "RefArchGlobal",
+                "l10n": {
+                    "supportedCurrencies": ["GBP"],
+                    "defaultCurrency": "GBP",
+                    "supportedLocales": [
+                        {
+                            "id": "en-GB",
```

Comment:
again, if URL settings have locale and site as `none`, we could skip this config. And if we have to have it, default should be en-US and only one locale should be configured.

Reply:
@alexvuong - would it be possible to skip this part of the config when the locale and site is `none`?